Definition for record field from template doesn't jump to Java source

Open https://github.com/ia3andy/quarkus-blast/blob/86d5abed4f6fac4adaf0ccf438124a8048671365/src/main/resources/templates/QuarkusBlast/index.html#L13

Try to hyperlink `score` from `{game.score}`, you will have have the following NPE

```
Internal error: java.lang.NullPointerException: Cannot invoke "com.intellij.openapi.editor.Document.getLineNumber(int)" because "document" is null

java.util.concurrent.CompletionException: java.lang.NullPointerException: Cannot invoke "com.intellij.openapi.editor.Document.getLineNumber(int)" because "document" is null
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:332)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:347)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:708)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2162)
	at com.redhat.devtools.intellij.lsp4ij.IndexAwareLanguageClient.runTask(IndexAwareLanguageClient.java:64)
	at com.redhat.devtools.intellij.lsp4ij.IndexAwareLanguageClient$1.run(IndexAwareLanguageClient.java:39)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:429)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:114)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressInCurrentThread$11(CoreProgressManager.java:557)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:186)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:173)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcessWithProgressInCurrentThread(CoreProgressManager.java:557)
	at com.intellij.openapi.progress.impl.CoreProgressManager.run(CoreProgressManager.java:363)
	at com.redhat.devtools.intellij.lsp4ij.IndexAwareLanguageClient.lambda$runAsBackground$0(IndexAwareLanguageClient.java:36)
	at com.redhat.devtools.intellij.lsp4ij.IndexAwareLanguageClient.lambda$runAsBackground$1(IndexAwareLanguageClient.java:50)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.NullPointerException: Cannot invoke "com.intellij.openapi.editor.Document.getLineNumber(int)" because "document" is null
	at com.redhat.devtools.intellij.lsp4ij.LSPIJUtils.toPosition(LSPIJUtils.java:155)
	at com.redhat.devtools.intellij.lsp4ij.LSPIJUtils.toRange(LSPIJUtils.java:176)
	at com.redhat.devtools.intellij.lsp4ij.LSPIJUtils.toLocation(LSPIJUtils.java:185)
	at com.redhat.devtools.intellij.lsp4mp4ij.psi.internal.core.ls.PsiUtilsLSImpl.toLocation(PsiUtilsLSImpl.java:107)
	at com.redhat.devtools.intellij.qute.psi.QuteSupportForTemplate.getJavaDefinition(QuteSupportForTemplate.java:205)
	at com.redhat.devtools.intellij.qute.lsp.QuteLanguageClient.lambda$getJavaDefinition$8(QuteLanguageClient.java:146)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:898)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:68)
	at com.redhat.devtools.intellij.qute.lsp.QuteLanguageClient.lambda$getJavaDefinition$9(QuteLanguageClient.java:146)
	at com.redhat.devtools.intellij.lsp4ij.IndexAwareLanguageClient.runTask(IndexAwareLanguageClient.java:60)
	... 22 more

--- a/src/main/java/com/redhat/devtools/intellij/lsp4ij/LSPIJUtils.java
+++ b/src/main/java/com/redhat/devtools/intellij/lsp4ij/LSPIJUtils.java
@@ -181,8 +181,10 @@ public class LSPIJUtils {
         if (sourceElement != null) {
             PsiFile file = sourceElement.getContainingFile();
             Document document = PsiDocumentManager.getInstance(psiMember.getProject()).getDocument(file);
-            TextRange range = sourceElement.getTextRange();
-            return toLocation(file, toRange(range, document));
+            if (document != null) {
+                TextRange range = sourceElement.getTextRange();
+                return toLocation(file, toRange(range, document));
+            }
         }
         return null;
     }

